package/cfg/rke2-cis-1.5/etcd.yaml

```diff
@@ -0,0 +1,135 @@
+---
+controls:
+version: 1.5
+id: 2
+text: "Etcd Node Configuration"
+type: "etcd"
+groups:
+- id: 2
+  text: "Etcd Node Configuration Files"
+  checks:
+    - id: 2.1
+      text: "Ensure that the --cert-file and --key-file arguments are set as appropriate (Scored)"
+      audit: "/bin/ps -ef | /bin/grep $etcdbin | /bin/grep -v grep"
+      tests:
+        bin_op: and
+        test_items:
+          - flag: "--cert-file"
+            set: true
+          - flag: "--key-file"
+            set: true
+      remediation: |
```

Comment:
Do we need any custom remediations for rke2 or the default will apply?

Reply:
Defaults should apply.